js/scripts.js

```diff
@@ -0,0 +1,20 @@
+$(document).ready(function() {
+	function Phone(brand ,color ,memory ,price) {
+		this.brand = brand;
+		this.price = price;
+		this.memory = memory;
+		this.color = color;
+	}
+	Phone.prototype.printInfo = function() {
+		console.log("The phone brand is " + this.brand + ", color is " + this.color + " memory: " + this.memory + " GB " + " and the price is " + this.price + ".");
+	}
+
+	var SamsungGalaxyS6 = new Phone("Samsung" ,"black" , 32 , 2000);
```

Comment:
variables names should start with small letter - first big letter is reserved only for constructor functions